google/cloud/forseti/scanner/scanners/service_account_key_scanner.py

```diff
@@ -0,0 +1,141 @@
+# Copyright 2017 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Scanner for the Service Account Keys."""
+
+from google.cloud.forseti.common.gcp_type.service_account import ServiceAccount
+from google.cloud.forseti.common.util import logger
+from google.cloud.forseti.scanner.audit import service_account_key_rules_engine
+from google.cloud.forseti.scanner.scanners import base_scanner
+
+LOGGER = logger.get_logger(__name__)
+
+
+class ServiceAccountKeyScanner(base_scanner.BaseScanner):
+    """Check if ServiceAccount Keys have not been rotated."""
+
+    def __init__(self, global_configs, scanner_configs, service_config,
+                 model_name, snapshot_timestamp, rules):
+        """Initialization.
+
+        Args:
+            global_configs (dict): Global configurations.
+            scanner_configs (dict): Scanner configurations.
+            service_config (ServiceConfig): Forseti 2.0 service configs
+            model_name (str): name of the data model
+            snapshot_timestamp (str): Timestamp, formatted as YYYYMMDDTHHMMSSZ.
+            rules (str): Fully-qualified path and filename of the rules file.
+        """
+        super(ServiceAccountKeyScanner, self).__init__(
+            global_configs,
+            scanner_configs,
+            service_config,
+            model_name,
+            snapshot_timestamp,
+            rules)
+
+        self.rules_engine = \
```

Comment:
Python style guide recommends using `()` for line continuation:
https://google.github.io/styleguide/pyguide.html?showone=Line_length#Line_length
```
self.rules_engine = (
    service_account_key_rules_engine.ServiceAccountKeyRulesEngine(
        rules_file_path=self.rules,
        snapshot_timestamp=self.snapshot_timestamp))

Reply:
Missed that recommendation - fixed!